### PR TITLE
Alert on LTI courses on UNCode platform

### LINKED ISSUE #442: Alert for LTI students that try to enter via UNCode Platform instead of LMS course
**Is your feature request related to a problem? Please describe.**
LTI students try to enter their tasks via UNCode main platform but if the logic behind doesn't allow this, they must enter via the LTI binding in the LMS platform, but in the main platform doesn't show the users that it isn't allowed.

**Describe the solution you'd like**
An alert, could be a modal or popup, that indicate, if an LTI user tries to enter via main platform, that they must enter via their LMS platform in order to see their task and make submissions.

### PR DESCRIPTION
**Pull request by VirtualDiegox**
# Description

Now on two pages, My courses and Courses list, the LTI courses that are displayed to students show an alert if they try to enter to these courses. This shows students that in order to access the tasks of the course they need to enter via their LMS platform such as Moodle or OpenEdx
image
image

Fixes #442 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

This was tested on my develop environment, whit two scenarios, one being admin of a LTI course and another being student of the same course, in the first case the admins can enter the courses without problem and in the second case if the student try to enter a modal alert is shown.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: inginious/frontend/templates/index.html [hunk 1]
```diff
@@ -37,19 +37,57 @@ <h2>$:_("Course list")</h2>
 <div class="list-group list-group-progress" style="margin-left: 16px; margin-right: 16px;">
     $ username = user_manager.session_username()
     $for courseid, course in open_courses.items():
+        $ is_lti_student = (course.is_lti() and not user_manager.user_is_superadmin() and (username not in course.get_admins()))
```
Comment (ferestrepoca) [inline inginious/frontend/templates/index.html:40]: Is it also necessary to discard course tutors (`course.get_tutors()`)?

Comment (ferestrepoca) [reply inginious/frontend/templates/index.html:40]: Please revise also the method `user_manager.has_admin_rights_on_course(course)`

**Review by ferestrepoca — CHANGES_REQUESTED** [review]

File: inginious/frontend/templates/index.html (hunk 1 above)
Comment (VirtualDiegox) [reply inginious/frontend/templates/index.html:40]: Tutors are not required for now, thanks

**Review by ferestrepoca — APPROVED** [review]
Thanks!!

**Pull request merged**